### PR TITLE
Docs fixes

### PR DESCRIPTION
**Pull request by samjhecht**
- renaming replicator tutorial
- fixing typos in JDBC tutorial (fixes https://github.com/confluentinc/cp-docker-images/issues/140)

### CODE REVIEW

**Comment by jonchiu** [comment]
Took a quick scan and LGTM.

File: docs/tutorials/replicator.rst [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
 It is worth noting that we will be configuring Kafka and Zookeeper to store data locally in the Docker containers.  For deployments that require persistent data (e.g. production deployments), you should use mounted volumes for persisting data in the event that a container stops running or is restarted.  This is important when running a system like Kafka on Docker, as it relies heavily on the filesystem for storing and caching messages. Refer to our `documentation on Docker external volumes <operations/external-volumes.html>`_ for an example of how to add mounted volumes to the host machine.
 
 Installing & Running Docker
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For this tutorial, we'll run docker using the Docker Compose.
+For this tutorial, we'll run Docker using the Docker Compose.
```
Comment (theduderog) [inline docs/tutorials/replicator.rst]: just say `using Docker Compose` instead of `using the Docker Compose`?

**Comment by theduderog** [comment]
Some wording nit picks and LGTM

**Comment by samjhecht** [comment]
Thanks, @theduderog .  I'm a little embarrassed that you had to catch those.

**Pull request merged**